integration-commons/src/main/java/org/symphonyoss/integration/service/KeyManagerService.java

```diff
@@ -0,0 +1,24 @@
+package org.symphonyoss.integration.service;
+
+import org.symphonyoss.integration.model.UserKeyManagerData;
+
+/**
+ * Contract to define responsibilities to provide key manager services.
+ * Created by campidelli on 9/11/17.
+ */
+public interface KeyManagerService {
+
+  /**
+   * Return the Key Manager data for the given user.
+   * @param userId The bot user ID.
+   * @return Key manager user data.
+   */
+  UserKeyManagerData getBotUserAccountKeyByUser(String userId);
```

Comment:
I think this method isn't required to be part of the interface.

Reply:
Done.